tests/manage/test_pvresize_with_node_down.py

```diff
@@ -0,0 +1,228 @@
+"""
+OCS-368
+"""
+import logging
+import pytest
+import json
+from time import sleep
+import threading
+
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import tier1, ManageTest
+from ocs_ci.ocs.resources.pvc import PVC
+from ocs_ci.ocs.resources.pod import Pod
+from ocs_ci.ocs.resources.pod import get_ceph_tools_pod
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.utility import templating
+
+log = logging.getLogger(__name__)
+
+OCS_BUG_ID = 'test-ocs-368'
+
+NAMESPACE = ocp.OCP(kind='namespace')
+OUR_PVC = None
+POD = None
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    Finalize teardown and call setup
+    """
+    def finalizer():
+        teardown()
+    request.addfinalizer(finalizer)
+    setup()
+
+
+def check_obj(obj_type):
+    """
+    Check pvc and pod.  Only one of each will be created in this test.
+
+    Return metadata
+    """
+    cmd_str = f'oc -n {config.ENV_DATA["my_namespace"]} get {obj_type} -o json'
```

Comment:
Not sure this is needed. Have a look in tests/helpers.py resource creation functions, validation is done as part of them

Reply:
I am doing this because on polarion, the report says to verify the creation  of the pvc and of the pod.  I realize that if the pvc or the pod does not get created, the test will fail anyway, so this check may be unnecessary.  I will remove it people  agree that this check is not needed.

Reply:
I think that we can remove this test.   The next version that I check in will not have it.